### PR TITLE
fix server error code match and add human readable opcode

### PR DESCRIPTION
**Pull request by bergwolf**

### CODE REVIEW

File: src/abi/linux_abi.rs [hunk 1]
```diff
@@ -640,12 +640,22 @@ pub enum Opcode {
...
     SetupMapping = 48,
     RemoveMapping = 49,
+    MaxOpcode = 50,
 
     /* Reserved opcodes: helpful to detect structure endian-ness in case of e.g. virtiofs */
     CuseInitBswapReserved = 1_048_576, /* CUSE_INIT << 8 */
     InitBswapReserved = 436_207_616,   /* FUSE_INIT << 24 */
 }
 
+impl Opcode {
```
Comment (jiangliu) [inline src/abi/linux_abi.rs:650]: Could we use
impl From<u32> for Opcode {
}
?

File: src/api/pseudo_fs.rs [hunk 2]
```diff
@@ -236,6 +236,10 @@ impl PseudoFs {
         let inodes = self.inodes.load();
 
         let inode = inodes.get(&ino).unwrap();
+        if ino == inode.parent {
```
Comment (jiangliu) [inline src/api/pseudo_fs.rs:239]: A comment is welcomed here:)

File: src/abi/linux_abi.rs (hunk 1 above)
Comment (bergwolf) [reply src/abi/linux_abi.rs]: sounds good!

File: src/api/pseudo_fs.rs (hunk 2 above)
Comment (bergwolf) [reply src/api/pseudo_fs.rs:241]: fixed!

File: src/abi/linux_abi.rs (hunk 1 above)
Comment (bergwolf) [reply src/abi/linux_abi.rs]: fixed

**Pull request merged**